src/KaliClubSig.sol

```diff
@@ -185,110 +185,110 @@ contract KaliClubSig is ClubNFT, Multicall, IClub {
     /// Operations
     /// -----------------------------------------------------------------------
 
+    struct Call {
+        address to;
+        uint256 value;
+        bytes data;
+        bool deleg;
+    }
+
     function execute(
-        address to,
-        uint256 value,
-        bytes memory data,
-        bool deleg,
+        Call[] calldata calls,
         Signature[] calldata sigs
     ) external payable returns (bool success) {
-        // governor has admin privileges to execute without quorum
-        if (!governor[msg.sender]) {
-            // cannot realistically overflow on human timescales
-            unchecked {
-                // TODO(Potential reentrancy bug here incrementing nonce before external calls)
-                // Consider intermediary storage on the stack and update after external calls
-                bytes32 digest = keccak256(
-                    abi.encodePacked(
-                        "\x19\x01",
-                        DOMAIN_SEPARATOR(),
-                        keccak256(
-                            abi.encode(
-                                keccak256(
-                                    "Exec(address to,uint256 value,bytes data,bool deleg,uint256 nonce)"
-                                ),
-                                to,
-                                value,
-                                data,
-                                deleg,
-                                ++nonce
+        for (uint256 i; i < calls.length; ) {
+            if (!governor[msg.sender]) {
```

Comment:
You could read this once outside of the loop, though I suspect the optimizer should also optimize it.

Reply:
thanks!